components/icon/style.module.css

```
@@ -1,7 +1,9 @@
-.icon {
-  fill: rgba(0, 0, 0, 0.26) !important;
+.icon :global(img) {
+  width: 24px;
+  height: 24px;
```

This should fix the Codacy Static Code Analysis errors.
```suggestion
  height: 24px;

```